bpfd/src/bpf.rs

```diff
@@ -48,13 +48,138 @@ const MAPS_MODE: u32 = 0o0660;
 
 pub(crate) struct BpfManager {
     config: Config,
-    dispatchers: HashMap<DispatcherId, Dispatcher>,
-    programs: HashMap<Uuid, Program>,
+    dispatchers: BpfDispatchers,
```

Comment:
I would name the new wrapper `DispatcherMap` since it's a map of dispatchers with some fancy helpers.

Reply:
Done